includes/settings/settings-ppec.php

```diff
@@ -521,6 +523,7 @@ function showHideDefaultButtonSettings() {
 		'class'       => 'wc-enhanced-select woocommerce_ppec_paypal_spb',
 		'default'     => 'rect',
 		'desc_tip'    => true,
+		'description' => __( 'The pill-shaped button\'s unique and powerful shape signifies PayPal in people’s minds. Use the rectangular button as an alternative when pill-shaped buttons might pose design challenges.', 'woocommerce-gateway-paypal-express-checkout' ),
```

Comment:
Intentional usage of both `'` and `’` ?

Reply:
Nice catch – changed in https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/409/commits/088b59cff622190aa7a2f413b5e55b579c2348a8.